sdk-dotnet/LittleHorse.Sdk/Helper/LHMappingHelper.cs

```diff
@@ -214,6 +214,25 @@ public static VariableType ValueCaseToVariableType(VariableValue.ValueOneofCase
             }
         }
 
+        /// <summary>
+        /// Maps a .NET <see cref="Type"/> to a workflow <see cref="ReturnType"/>.
+        /// </summary>
+        /// <param name="type">The .NET type to map.</param>
+        /// <returns>The corresponding workflow <see cref="ReturnType"/>.</returns>
+        /// <exception cref="ArgumentException">Thrown if the type is not supported.</exception>
+        public static ReturnType DotNetTypeToReturnType(Type? type)
+        {
+            if (type == null)
+            {
+                throw new ArgumentNullException(nameof(type),"Type cannot be null.");
+            }
+            var typeDef = new TypeDefinition
+            {
+                Type = DotNetTypeToLHVariableType(type!)
```

Comment:
Eventually, we should deprecate / remove the `DotNetTypeToLHVariableType()` method and replace it with one that returns a `TypeDefinition` instead of a `VariableType`. This will enable us to support